src/postman/__tests__/operation.test.ts

```diff
@@ -283,3 +283,40 @@ describe('transformPostmanCollectionOperation()', () => {
     });
   });
 });
+
+describe('transformPostmanCollectionOperations()', () => {
+  it('transforms operations nested in folders', () => {
+    const operations = transformPostmanCollectionOperations({
+      item: [
+        {
+          item: [
+            { request: { method: 'get', url: '/a/a' } },
+            {
+              item: [{ request: { method: 'get', url: '/a/a/a' } }, { request: { method: 'get', url: '/a/a/b' } }],
+            },
+          ],
+        },
+        { request: { method: 'get', url: '/a' } },
+      ],
+    });
+
+    expect(operations).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          path: '/a/a',
+        }),
+        expect.objectContaining({
+          path: '/a/a/a',
+        }),
+        expect.objectContaining({
+          path: '/a/a/b',
+        }),
+        expect.objectContaining({
+          path: '/a',
+        }),
+      ]),
+    );
+
+    expect(operations).toHaveLength(4);
```

Comment:
This statement is already implicit in the first expect. Fine to leave it, just saying.

Reply:
I think it is not. The resulting array may be longer than 4 and that is what we also want to avoid. See `expect.arrayContaining`

Reply:
That's true.